### PR TITLE
Fix reference data relation import

### PR DESCRIPTION
**Pull request by LeoBenoist**
Fix:

08:33:42 ERROR     [console] Error thrown while running command "--env=prod akeneo:batch:job -c '{"filePath":"/import\/product_model_clothing_colorsize.csv"}' csv_product_model_import". Message: "Call to undefined method MyProject\Bundle\CatalogBundle\Entity\Supplier::getReference()" ["error" => Error { …},"command" => "--env=prod akeneo:batch:job -c '{"filePath":"/import\/product_model_clothing_colorsize.csv"}' csv_product_model_import","message" => "Call to undefined method MyProject\Bundle\CatalogBundle\Entity\Supplier::getReference()"] []

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added Behats                      | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Comment by LeoBenoist** [comment]
It seem getReference is something necessary in order to have RefData working during import of product model with ref data. You may want to implement an interface

**Comment by damien-carcel** [comment]
Hi Leo. Thank you for the PR, but your branch destination is wrong. Can you change it for the `2.0` please (and of course rebase your branch on it)?

File: src/Pim/Component/ReferenceData/Model/AbstractReferenceData.php [hunk 1]
```diff
@@ -63,6 +63,11 @@ public function setSortOrder($sortOrder)
 
         return $this;
     }
+    
+    public function getReference()
```
Comment (solivier) [inline src/Pim/Component/ReferenceData/Model/AbstractReferenceData.php]: Don't forget the doc, and maybe you should add it into the interface.

Comment (damien-carcel) [reply src/Pim/Component/ReferenceData/Model/AbstractReferenceData.php]: Not maybe imo :-)

**Comment by damien-carcel** [comment]
I will not put a GTM on this PR yet (nor block it), because I think you should not need this method in any cases: we have product imports with reference data in our behat test suites (`features/reference-data/import/import_products.feature`) which are using the entities defined in the AcmeAppBundle (in `src/Acme/Bundle/AppBundle/Entity/Color.php` and `src/Acme/Bundle/AppBundle/Entity/Fabric.php`).

These two entities do not implement the `getReference` method, and yet the imports are working, so I think they are not mandatory.

I also see we use this method for the Assets in EE, but I honnestly don't know why we need it in this case. @jjanvier wrote this entity, so maybe he knows.

**Pull request closed without merging**